### PR TITLE
[6.3] FIX UI  test_positive_latest_warning_error_tasks

### PR DESCRIPTION
**Pull request by ldjebran**
the task id field was moved to task raw tab and the error name was with leading space " error"
```console
(sat-6.3.0) dlezz@elysion:~/projects/robottelo-fork$ py.test tests/foreman/ui/test_dashboard.py -v -k  "test_positive_latest_warning_error_tasks"
================================================= test session starts ==================================================
platform linux2 -- Python 2.7.13, pytest-3.0.7, py-1.4.34, pluggy-0.4.0 -- /home/dlezz/.pyenv/versions/sat-6.3.0/bin/python2.7
cachedir: .cache
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.0, cov-2.4.0
collected 25 items 
2017-07-07 17:34:32 - conftest - DEBUG - Found WONTFIX in decorated tests ['1269196', '1378009', '1245334', '1217635', '1226425', '1156555', '1199150', '1204686', '1267224', '1221971', '1103157', '1230902', '1214312', '1079482']

2017-07-07 17:34:32 - conftest - DEBUG - Collected 25 test cases


tests/foreman/ui/test_dashboard.py::DashboardTestCase::test_positive_latest_warning_error_tasks <- ../../.pyenv/versions/sat-6.3.0/lib/python2.7/site-packages/robozilla/decorators/__init__.py PASSED

================================================= 24 tests deselected ==================================================
======================================= 1 passed, 24 deselected in 39.44 seconds =======================================
```

### CODE REVIEW

File: robottelo/ui/dashboard.py [hunk 1]
```diff
@@ -183,12 +183,18 @@ def validate_error_navigation(
         """
         self.navigate_to_entity()
         self.click(locators['dashboard.lwe_task.name'] % task_name)
+        redirection_error_msg = ('Redirection to task details page does not'
```
Comment (oshtaier) [inline robottelo/ui/dashboard.py]:
```
self.navigate_to_entity()
self.click(locators['dashboard.lwe_task.name'] % task_name)
if self.wait_until_element(tab_locators['task.tab_task']) is None:
    raise UIError('Redirection to task page does not work properly')
if expected_result:
    actual_result = self.wait_until_element(locators['task.selected.result']).text.strip()
    if actual_result != expected_result:
        raise UIError('Task finished with unexpected result')
...
```

**Comment by oshtaier** [comment]
and don't forget to verify defect and put corresponding state for the BZ
https://bugzilla.redhat.com/show_bug.cgi?id=1460240

**Comment by ldjebran** [comment]
```console
(sat-6.3.0) dlezz@elysion:~/projects/robottelo-fork$ py.test tests/foreman/ui/test_dashboard.py -v -k  "test_positive_latest_warning_error_tasks"
================================================= test session starts ==================================================
platform linux2 -- Python 2.7.13, pytest-3.0.7, py-1.4.34, pluggy-0.4.0 -- /home/dlezz/.pyenv/versions/sat-6.3.0/bin/python2.7
cachedir: .cache
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.0, cov-2.4.0
collected 25 items 
2017-07-10 18:28:07 - conftest - DEBUG - Found WONTFIX in decorated tests ['1269196', '1378009', '1245334', '1217635', '1226425', '1156555', '1199150', '1204686', '1267224', '1221971', '1103157', '1230902', '1214312', '1079482']

2017-07-10 18:28:07 - conftest - DEBUG - Collected 25 test cases


tests/foreman/ui/test_dashboard.py::DashboardTestCase::test_positive_latest_warning_error_tasks <- ../../.pyenv/versions/sat-6.3.0/lib/python2.7/site-packages/robozilla/decorators/__init__.py PASSED

================================================= 24 tests deselected ==================================================
======================================= 1 passed, 24 deselected in 53.01 seconds =======================================
```

**Comment by ldjebran** [comment]
@oshtaier comment addressed

**Comment by oshtaier** [comment]
*and active*

**Comment by ldjebran** [comment]
we can also get the attribute aria-expanded of ta_tab_element to check that it's equal to true but think it's necessary as soon as we have clicked on it
```console
======================================= 1 passed, 24 deselected in 57.63 seconds =======================================
(sat-6.3.0) dlezz@elysion:~/projects/robottelo-fork$ py.test tests/foreman/ui/test_dashboard.py -v -k  "test_positive_latest_warning_error_tasks"
================================================= test session starts ==================================================
platform linux2 -- Python 2.7.13, pytest-3.0.7, py-1.4.34, pluggy-0.4.0 -- /home/dlezz/.pyenv/versions/sat-6.3.0/bin/python2.7
cachedir: .cache
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.0, cov-2.4.0
collected 25 items 
2017-07-12 18:11:15 - conftest - DEBUG - Found WONTFIX in decorated tests ['1269196', '1378009', '1245334', '1217635', '1226425', '1156555', '1199150', '1204686', '1267224', '1221971', '1103157', '1230902', '1214312', '1079482']

2017-07-12 18:11:15 - conftest - DEBUG - Collected 25 test cases


tests/foreman/ui/test_dashboard.py::DashboardTestCase::test_positive_latest_warning_error_tasks <- ../../.pyenv/versions/sat-6.3.0/lib/python2.7/site-packages/robozilla/decorators/__init__.py PASSED

================================================= 24 tests deselected ==================================================
======================================= 1 passed, 24 deselected in 54.99 seconds =======================================
```
@oshtaier comment addressed

**Pull request merged**